erts/etc/common/erlexec.c

```diff
@@ -554,38 +554,54 @@ int main(int argc, char **argv)
 
     if (s == NULL) {
         erts_snprintf(tmpStr, sizeof(tmpStr),
-            "%s" PATHSEP "%s" DIRSEP "bin" PATHSEP, bindir, rootdir);
+            "%s" PATHSEP "%s" DIRSEP "bin", bindir, rootdir);
```

Comment:
We removed the trailing the PATHSEP here since the PATH should never end with `:` afaik.